spec/fabricators/keyword_fabricator.rb

```diff
@@ -5,3 +5,15 @@
   name { FFaker::FreedomIpsum.word }
   user { Fabricate(:user) }
 end
+
+Fabricator(:keyword_parsed, from: :keyword) do
+  status :parsed
+  ads_top_count { FFaker.rand 9 }
+  ads_page_count { FFaker.rand 9 }
+  ads_top_urls(rand: 9) { FFaker::Internet.http_url }
+  result_count { FFaker.rand 9 }
+  result_urls(rand: 9) { FFaker::Internet.http_url }
+  total_link_count { FFaker.rand 9 }
+  html { FFaker::HTMLIpsum.body }
+  source { Fabricate(:source) }
```

Comment:
```suggestion

  source { Fabricate(:source) }
```

Let's try to separate the associated fields from the direct fields. This improves the readability ~